packages/editor/src/store/test/actions.js

```diff
@@ -62,169 +36,6 @@ describe( 'actions', () => {
 			} );
 		} );
 	} );
-	describe( 'resetBlocks', () => {
```

Comment:
We still have `resetEditorBlocks`, which dispatches the same `RESET_BLOCKS` action tested here. Why was it renamed? Why wasn't the action type renamed accordingly? Why was the test case removed?

Reply:
Because of the proxyfying needed for BC, we already have a `resetBlocks` actions targetting the core/block-editor one and this `resetEditorBlocks` is very different from the original one. That's why initially I removed all these tests as they moved to the block-editor module.

We can add tests back for this new action/reducer... but these are still new.

Reply:
> Because of the proxyfying needed for BC, we already have a `resetBlocks` actions targetting the core/block-editor one and this `resetEditorBlocks` is very different from the original one. That's why initially I removed all these tests as they moved to the block-editor module.
> 
> We can add tests back for this new action/reducer... but these are still new.

I'd suggest we rename the action type as well to keep it in sync with the name of the action creator.

Reply:
Renamed, I'm still hesitant here. I feel this action should be `updateEditorBlocks` as "reset" means something happening from time to time resetting the whole editor to something completely different while in reality, this action is triggered for all block changes, it's more an "update" than a "reset" to me.